DRAFT.md

```diff
@@ -638,15 +641,30 @@ constexpr auto operator<=>(const U& lhs, const indirect<T, A>& rhs);
 * _Remarks_: Specializations of this function template for which `*lhs <=> *rhs`
   is a core constant expression are constexpr functions.
 
-#### Allocator related traits
+#### X.Y.10 Allocator related traits [indirect.allocator.traits]
 
 ```c++
 template <class T, class Alloc>
-struct std::uses_allocator<xyz::indirect<T>, Alloc> : true_type {};
+struct std::uses_allocator<indirect<T>, Alloc> : true_type {};
 ```
 
 * _Preconditions_: Alloc meets the _Cpp17Allocator_ requirements.
 
+#### X.Y.11 Hash support [indirect.hash]
+  
+```c++
+template <class T, class Alloc>
+struct std::hash<indirect<T, Alloc>>;
+```
+
+* _Preconditions_: `i` is not valueless.
+
+The specialization `hash<indirect<T>>` is enabled ([unord.hash]) if and only if
```

Comment:
In this paragraph you mention `indirect<T>` three times. But it probably should say `indirect<T, Alloc>` instead?

Reply:
Thanks, well-spotted. I will amend.